src/main/java/org/example/utils/JwtUtils.java

```diff
@@ -0,0 +1,36 @@
+package org.example.utils;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.security.Keys;
+
+import javax.crypto.SecretKey;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JwtUtils {
+    // Generate a SecretKey for HS256
```

Comment:
do we need this comment? both secret generation and the algorithm is clear from the statement